src/views/DatasetManager/components/CreateDatasetButton/components/DatasetCreationParameters/DatasetCreationParameters.js

```diff
@@ -126,6 +128,19 @@ export const DatasetCreationParameters = ({ dataSourceRunTemplates, parentDatase
                   />
                 </Grid>
               );
+            } else if (inputType === 'date') {
+              return (
+                <Grid container columns={6} sx={{ pt: 1 }}>
+                  <GenericDateInput
+                    parameterData={parameter}
+                    context={{ editMode: true }}
+                    parameterValue={value}
+                    setParameterValue={onChange}
+                    isDirty={null}
+                    error={null}
```

Comment:
I'm not sure that it can be important, but we don't have any error handling and if we are putting a date in wrong format, e.g. dd/mm/yyyy and not mm/dd/yyyy, the date just disappears :woman_shrugging: but if error handling isn't expected by the end-user, it's ok :wink:

Reply:
At this state, we don't handle error in dataset manager.